com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/test/PlatformTest.java

```diff
@@ -68,6 +71,18 @@ public void testPlatformOS() {
         assertNull(Platform.get(""));
     }
 
+    @Test
+    public void testPlatformArchitectures() {
+        for (Platform platform : Platform.values())
+        {
+            List<String> availableArchitectures = Arrays.asList(platform.getArchitectures().getArchitectures());
+            if (!availableArchitectures.contains(platform.getPair()))
+            {
+                System.out.println(String.format("ERROR! %s is not a supported architecture for %s platform. Available architectures: %s", platform.getPair(), platform.getPair(), String.join(", ", availableArchitectures)));
+                assertTrue(false);
+            }
+        }
+    }
```

Comment:
Loop through all Platform types and make sure they're represented in the architectures (same as Bob does)